### PR TITLE
Issue #3782: Add %D and %{R}W to NCSA Logging Examples

### PR DESCRIPTION
**Pull request by kgibm**

### CODE REVIEW

File: modules/ROOT/pages/access-logging.adoc [hunk 1]
```diff
@@ -142,3 +142,5 @@ Based on this setting, the NCSA access log will include the following informatio
 * The first line of the request
 * The status code of the response
 * The response size in bytes excluding headers
+* The elapsed time of the request in milliseconds (end-to-end including client and network time)
```
Comment (dmuelle) [inline modules/ROOT/pages/access-logging.adoc]: revise to remove parentheses per ID guidelines:

The elapsed time of the request in milliseconds, end-to-end, including client and network time

File: modules/ROOT/pages/access-logging.adoc [hunk 2]
```diff
@@ -142,3 +142,5 @@ Based on this setting, the NCSA access log will include the following informatio
...
+* The elapsed time until the first bytes of the response are sent in milliseconds (a close approximation of application response time)
```
Comment (dmuelle) [inline modules/ROOT/pages/access-logging.adoc]: The elapsed time until the first bytes of the response are sent in milliseconds. This value is a close approximation of application response time.

File: modules/ROOT/pages/access-logging.adoc [hunk 3]
```diff
@@ -48,15 +48,15 @@ Use the `accessLogFormat` property to specify the information  and format you wa
 This log format string is specified by using the `logFormat` attribute of `httpAcccessLogging` or `accessLogging` elements in the `server.xml` file, as shown in the following examples:
 
 ----
-<httpAccessLogging logFormat='%h %u %{t}W "%r" %s %b'/>
+<httpAccessLogging logFormat='%h %u %{t}W "%r" %s %b %D %{R}W'/>
```
Comment (dmuelle) [inline modules/ROOT/pages/access-logging.adoc:51]: To clarify- is `%{R}W` an actual attribute  value or does `{R}` represent a variable that would change depending on the config details

**Review by dmuelle — CHANGES_REQUESTED** [review]
looks good, just a question for clarification and few minor changes per ID guidelines

File: modules/ROOT/pages/access-logging.adoc (hunk 3 above)
Comment (kgibm) [reply modules/ROOT/pages/access-logging.adoc:51]: `%{R}W` is [the actual attribute value](https://openliberty.io/docs/latest/access-logging.html#_http_access_log_format):

> %{R}W | Service  time of the request from the moment the request is received until the  first set of bytes of the response is sent - millisecond accuracy,  microsecond precision

File: modules/ROOT/pages/access-logging.adoc (hunk 1 above)
Comment (kgibm) [reply modules/ROOT/pages/access-logging.adoc]: I also just realized that the value is actually in microseconds (although millisecond accuracy), so I'll change milliseconds to microseconds.

File: modules/ROOT/pages/access-logging.adoc [hunk 4]
```diff
@@ -142,3 +142,5 @@ Based on this setting, the NCSA access log will include the following informatio
 * The first line of the request
 * The status code of the response
 * The response size in bytes excluding headers
+* The elapsed time of the request in microseconds, end-to-end, including client and network time
+* The elapsed time until the first bytes of the response are sent in microseconds. This value is often a close approximation of application response time.
```
Comment (dmuelle) [inline modules/ROOT/pages/access-logging.adoc]: The elapsed time until the first bytes of the response are sent in microseconds.
-->
The elapsed time in microseconds until the first bytes of the response are sent.

**Review by dmuelle — COMMENTED** [review]
sorry, one more small fix I should've caught the first time through

File: modules/ROOT/pages/access-logging.adoc (hunk 4 above)
Comment (kgibm) [reply modules/ROOT/pages/access-logging.adoc]: Sure, pushed a new commit

**Review by dmuelle — APPROVED** [review]

**Comment by kgibm** [comment]
@mrsaldana Hey Manuel, can you please review this doc change? This is adding `%{R}W` to the HTTP access logging examples. Here are the draft updates: https://docs-draft-openlibertyio.mybluemix.net/docs/21.0.0.3/access-logging.html

Here is a test:

```
$ docker run --rm openliberty/open-liberty:full-java8-ibmjava-ubi sh -c "curl https://tomcat.apache.org/tomcat-10.0-doc/appdev/sample/sample.war -so /config/dropins/sample.war && echo '<?xml version=\"1.0\" encoding=\"UTF-8\"?><server><httpEndpoint id=\"defaultHttpEndpoint\" host=\"*\" httpPort=\"9080\" httpsPort=\"9443\"><accessLogging filepath=\"\${server.output.dir}/logs/access.log\" logFormat=\"%h %u %{t}W &quot;%r&quot; %s %b %D %{R}W\" /></httpEndpoint></server>' > /config/configDropins/overrides/server.xml && (/opt/ol/wlp/bin/server run defaultServer &) && sleep 5 && tail -f -n0 /logs/messages.log | grep -q CWWKF0011I && curl -sv --trace-time http://localhost:9080/sample/hello && cat /opt/ol/wlp/output/defaultServer/logs/access.log && /opt/ol/wlp/bin/server stop defaultServer"
[...]
127.0.0.1 - [09/Mar/2021:20:23:05 +0000] "GET /sample/hello HTTP/1.1" 200 311 81140 76313
```

**Comment by mrsaldana** [comment]
The doc team already reviewed, but looks good to me too!

**Pull request merged**